lib/cmd/common/import.js

```diff
@@ -122,12 +124,21 @@ function importAppNgui(projectId, appTitle, appTemplateType, fileOrRepo, deployE
       connections:[],
       template: {
         repoUrl: repo,
-        type: appTemplateType
+        type: appTemplateType,
+        imported: true,
+        repoBranch: 'refs/heads/' + repoBranch
       }
     };
 
     var url = 'box/api/projects/' + projectId + '/apps';
-    common.doApiCall(fhreq.getFeedHenryUrl(), url, payload,  i18n._("Error importing git repo: "), cb);
+    common.doApiCall(fhreq.getFeedHenryUrl(), url, payload, i18n._("Error importing git repo: "), function(err, data) {
+      if (err) {
+        return cb(err);
+      }
+
+      log.silly('import response', data);
+      return cb(null, 'Import Successful.');
```

Comment:
@trepel I noticed this after I raised another few PR's. I think it should just return the JSON here like it does with other commands. I forgot that FHC can be used as a Node.js module and for some automation. Users probably need that JSON output. Reverting now.